src/styles/index.scss

```diff
@@ -1,3 +1,73 @@
-body {
+@import './utils/variables';
+
+.page {
   margin: 0;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
```

Comment:
remember about BEM rules (take into account next time) - all blocks should be in separate files